src/elements/Segment/Segments.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+import META from 'src/utils/Meta';
+import {childrenOfTypesOnly} from '../../utils/customPropTypes';
```

Comment:
Our app allows importing from the project root.  Suggest refactoring this util import to `src/utils/customPropTypes`.